.github/workflows/docsbuild-release.yaml

```diff
@@ -0,0 +1,78 @@
+name: ansible-release-porting-guide
+on:
+  workflow_dispatch:
+    inputs:
+      ansible-version:
+        description: >-
+          Release Version. Example: 11.1.0
+        required: true
+env:
+  CI_COMMIT_MESSAGE: 'Add Ansible community "${{ inputs.ansible-version }}" porting guide'
+
+jobs:
+  build:
+    name: 'Build Porting-Guide (${{ inputs.ansible-version }})'
+    runs-on: ubuntu-latest
+
+    steps:
+
+      - name: Retrieve the major version component
+        id: compute-version
+        run: |
+          echo -n 'major-version=' >> "${GITHUB_OUTPUT}"
+          echo '${{ inputs.ansible_version }}' | sed 's#\([0-9]\+\)\..*#\1#' >> "${GITHUB_OUTPUT}"
+
+      - name: Check out ansible-documentation
+        uses: actions/checkout@v3
+        with:
+          path: ansible-documentation
+          ref: devel
+
+      - name: Checking out to a new branch & setting the user details
+        working-directory: ansible-documentation
+        run: |
+          git checkout -b "publish-${{ inputs.ansible-version }}"
+          git config --global user.name "Github Actions"
+          git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
+
+# Get the Porting guide from the ansible-build-data git repo
+
+      - name: Check out ansible-build-data
+        uses: actions/checkout@v3
+        with:
+          repository: anweshadas/ansible-build-data
+          ref: main
+          path: ansible-build-data
```

Comment:
So this PR assumes that the PR in the ansible-build-data repository has already been merged?

Reply:
I think this was the agreement initially.

Reply:
Yes, it assumes that the build-data PR has already been merged.

Reply:
@anweshadas do we want to validate that the downloaded data contains the version we expect?
Maybe, run a simple `ls -1 ansible-build-data/ansible-${{ inputs.ansible-version }}-tags.yaml` to fail early?

Reply:
@felixfontein with my new suggestion above, this should be more obvious and accurate now: https://github.com/ansible/ansible-documentation/pull/121/files#r1572486604.